README.md

```diff
@@ -1,78 +1,161 @@
 
-Start the Docker image:
+Pull & start the Docker image:
 ```
+docker pull moov/imagecashletter:latest
 docker run -p 8083:8083 -p 9093:9093 moov/imagecashletter:latest
 ```
 
-List files stored in-memory
+List files stored in-memory:
 ```
 curl localhost:8083/files
 ```
 ```
-{"files":[],"error":null}
+null
+```
+
+Create a file on the HTTP server:
+```
+curl -XPOST --data-binary "@./test/testdata/valid-ascii.x937" http://localhost:8083/files/create
+```
+```
+{"id":"<YOUR-UNIQUE-FILE-ID>","fileHeader":{"id":"","standardLevel":"03","testIndicator":"T","immediateDestination":"061000146","immediateOrigin":"026073150", ...
 ```
 
-Create a file on the HTTP server
+Read the X9 file (in JSON form):
 ```
-curl -XPOST --data-binary "@./test/testdata/BNK20180905121042882-A.icl" http://localhost:8083/files/create
+curl http://localhost:8083/files/<YOUR-UNIQUE-FILE-ID>
 ```
 ```
-{"id":"71ae3f5bc5527cdb1efc88e1814333fd9d6d2edb","fileHeader":{"id":"","standardLevel":"35","testIndicator":"T","immediateDestination":"231380104","immediateOrigin":"121042882", ...
+{"id":"<YOUR-UNIQUE-FILE-ID>","fileHeader":{"id":"","standardLevel":"03","testIndicator":"T","immediateDestination":"061000146","immediateOrigin":"026073150", ...
 ```
 
-Create a file with the JSON format on the HTTP server
+Create a file with JSON format on the HTTP server:
 ```
 curl -XPOST -H "content-type: application/json" localhost:8083/files/create --data @./test/testdata/icl-valid.json
 ```
 ```
-{"id":"8afcde4fc2cf4023e92a7a96be9dbbe44e1e9508","fileHeader":{"id":"","standardLevel":"35","testIndicator":"T","immediateDestination":"231380104","immediateOrigin":"121042882", ...
+{"id":"<YOUR-UNIQUE-FILE-ID>","fileHeader":{"id":"","standardLevel":"35","testIndicator":"T","immediateDestination":"231380104","immediateOrigin":"121042882", ...
 ```
 
-Get the formatted file
+Get the formatted file:
 ```
-curl localhost:8083/files/8afcde4fc2cf4023e92a7a96be9dbbe44e1e9508/contents
+curl localhost:8083/files/<YOUR-UNIQUE-FILE-ID>/contents
 ```
 ```
-0135T231380104121042882201810032219NCitadel           Wells Fargo        US
-0123138010412104288220181003201810032219IGA1      Contact Name  5558675552
-0123138010412104288220181003201810039999      1   01
-      123456789 031300012             555888100001000001              GD1Y030B
-1121042882201810031              938383            01   Test Payee     Y10
+P0135T231380104121042882201810032219NCitadel      Wells Fargo    US   P100123138010412104288220181003201810032219IGA1   Contact Name 5558675552  P200123138010412104288220181003201810039999   1  01             P25   123456789 031300012       555888100001000001       GD1Y030BP261121042882201810031       938383      01  Test Payee   Y10
 ...
 ```
 
-### Go library
+### Google Cloud Run
```

Comment:
@adamdecaf This section looks good to me, but you should probably double check since you're more familiar with GCP.

Reply:
I don't know. We just had the Google Cloud Run button for ACH. Can we try this prior to merge?

Reply:
I can try it out. I've got a GCP account.

Reply:
I was going to use the run button originally, but the port number (8083) can't be defined through the button URL, so I went for a longer workaround.

Reply:
Looks like you can supply the port in configuration file: https://github.com/GoogleCloudPlatform/cloud-run-button#customizing-deployment-parameters 🤷‍♂️

Reply:
I tested out the steps as written and they worked great. I had never deployed an app to GCP, so I'd say this was a good test.

Reply:
Ok awesome, thanks for going through them.